install/azure_database/01_configure_azure_instance.mdx

```diff
@@ -11,25 +11,29 @@ export const ImgAzureServerParams = () => <img src={imgAzureServerParams} />
 
 
 In order to set up pganalyze monitoring for [Azure Database for PostgreSQL](https://azure.microsoft.com/en-us/products/postgresql/) you'll need
-to first follow these steps to enable the [pg\_stat\_statements](http://www.postgresql.org/docs/current/static/pgstatstatements.html)
+to first follow these steps to enable the [pg_stat_statements](http://www.postgresql.org/docs/current/static/pgstatstatements.html)
```

Comment:
?

Reply:
I believe this question mark is about what you are talking in the overall review comment, if it's okay to remove the escape (underscore). I checked this both in public and in app, both look okay without having the escapes.
Here is one in-app example that we already use "pg_stat_statements" without escaping (nor in the code block), but doing fine: https://staging.pganalyze.com/organizations/pganalyze/servers/install/aiven/01_create_monitoring_user

(I also mentioned this in https://github.com/pganalyze/pganalyze-docs/pull/230#discussion_r1548859435, hehe)

Reply:
Huh, okay. Good to know.